src/robot2017/src/robot_exec.cpp

```diff
@@ -0,0 +1,137 @@
+#include "robot_exec.h"
+#include "robot_msgs/Teleop.h"
+#include "motors.h"
+#include <ros/ros.h>
+#include <std_msgs/Int64.h>
+#include <string>
+
+
+RobotExec::RobotExec()
+{
+    bool dead = true;
+    bool autonomyActive = false;
```

Comment:
yeah that's all you would need - i fixed in the latest commit though using member initialization list